### ISSUE TITLE
Reorder: top button labels are not descriptive to communicate their purpose (2.4.6)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#956]
### Expected behavior
The buttons have descriptive labels which inform someone what to expect if clicked - the item in that row is moved to the top/beginning of the list. The label reads out "Move Title 1 to top of list" - Title 1 is the title of the item being moved.  

https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-descriptive.html
https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G131 

### Actual behavior
Each button is labeled "top" which doesn't communicate its purpose to someone who isn't looking at page and understands the spatial relationships and there is a list. No semantic structure for a ordered list is used so someone not looking at it wouldn't know it's a list.
